### DISCUSSION TITLE
【求助】取丁尼的时候看到好几个蒙面少女，我会不会寄啊？

### DISCUSSION THREAD

**Discussion by LBK-miq** [share121/inter-knot#2894] [其他]
![qq_pic_merged_1723997588292](https://github.com/user-attachments/assets/21edd2d7-94a9-4770-94bb-80940721034e)

如题，今天出门取丁尼，取出来了正出门呢，马上就看到好几个蒙面少女死盯着银行大门，看着就不怀好意，看见我了还在嘀咕些什么，这是怎么了？不是冲我来的吧？我会不会寄？绳网好心人能不能给个说法？？？急急急！！！<img src="https://upload-bbs.miyoushe.com/upload/2023/01/17/d1dd2f84c4b22880f119fb2151a33eba_4066649589662777644.png" alt="惊吓呜呜" width="75" >

**Comment by 9baka99** [comment]
没关系的吧，新艾利都的子弹伤害不如近战的
